sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/CoverageAggregator.java

```diff
@@ -47,8 +47,10 @@
 
   public CoverageAggregator(CoverageConfiguration coverageConf, Configuration configuration, FileSystem fs) {
 
-    Predicate<String> isSupportedLanguage = absolutePath -> fs.hasFiles(fs.predicates().and(fs.predicates().hasAbsolutePath(absolutePath),
-      fs.predicates().hasLanguage(coverageConf.languageKey())));
+    Predicate<String> isSupportedLanguage = absolutePath -> fs.hasFiles(
+      fs.predicates().and(
```

Comment:
this is just to make it more readable, because it:
- checks the language
- checks if the file is indexed (if not , it will fail)

probably it deserves a better name accross the code base